pybamm/solvers/scikits_dae_solver.py

```diff
@@ -14,6 +14,11 @@
     if scikits_odes_spec is not None:
         scikits_odes = importlib.util.module_from_spec(scikits_odes_spec)
         scikits_odes_spec.loader.exec_module(scikits_odes)
+        # NOTE: Gives error module 'scikits.odes' has no attribute 'sundials'
+        # if you try to make the Jacobian class using
+        # scikits_odes.sundials.ida.IDA_JacRhsFunction
+        # but is OK if you import ida here?
+        from scikits.odes.sundials import ida
```

Comment:
Weird, do other things like:
```python
import scikits.odes.sundials as sundials
...
sundials.ida
```
also break?

Reply:
I have no idea though. Probably need @martinjrobins

Reply:
depends how the scikits.odes ppl have set it up. You might not normally need scikits.odes.sundials.ida, so it might not all be brought in when you import scikits.odes. It might be purposeful (so you only import what you need), or the authors just forgot to import it. I think some submodules of scipy do that as well (can't think offhand which ones)